doc/OneStepSetup.md

```diff
@@ -12,6 +12,8 @@ This is a streamlined process for setting up the Pi. You'll flash a preconfigure
 
 1. Flash the [latest image release](https://github.com/marcone/teslausb/releases) using [Raspberry Pi Imager](https://www.raspberrypi.com/software/) or a similar flashing tool.
 
+    In Raspberry Pi Imager, you need to click 'Operating System' and then scroll _all the way down_ and select the 'Use custom' option.
```

Comment:
I assumed everybody just used drag&drop. Seems easier.

Reply:
oh. or that. But I feel some guidance is needed at this point :)

Reply:
Erm, no such thing as drag/drop on my Mac until you open the file picker though...

Reply:
What do you mean "no such thing as drag&drop"? Can't you just pick up a file and drop it onto the Raspberry Pi Imager window?

Reply:
No 😂

Reply:
oh, I see what you mean. Same on Windows. I wonder if that changed, or if I'm just misremembering

Reply:
Should I go ahead and open a drag and drop PR for https://github.com/raspberrypi/rpi-imager now? 😂

Reply:
Does macos have an "open with..." menu that lets you open the image file with Raspberry Pi Imager? That works for me on Windows (and might be worth mentioning as an alternative to using OS->Custom)

Reply:
Tried. I needed to first do 'All Applications' in the app picker (it defaults to 'Recommended Apps' which RPi Imager is not a part of) but then it doesn't work. Opened an Issue on the rpi imager repo about drag and drop.